README.md

```diff
@@ -1,8 +1,8 @@
-# Cloud.gov post mortems
+# cloud.gov post mortems
 
-**Please note - if you are experiencing a problem or want to discuss an ongoing incident, please visit #cloud-gov-support in Slack, or visit http://cloudgov.statuspage.io/. This repository and wiki is *only* for post-mortems after the incident has been closed.**
+**Note: If you're experiencing a problem with cloud.gov or you want to discuss an ongoing incident: check [the cloud.gov StatusPage](http://cloudgov.statuspage.io/) and visit #cloud-gov-support in Slack or email cloud-gov-support@gsa.gov. This repository and wiki are *only* for post-mortems after the incident has been closed.**
 
-A post mortem is held as soon as possible after an incident. ITIL defines an incident as `Failure of a configuration item that has not yet affected service is also an incident — for example, failure of one disk from a mirror set. The ITIL incident management process ensures that normal service operation is restored as quickly as possible and the business impact is minimized.'
+We hold a post mortem as soon as possible after a cloud.gov service disruption or other incident. We use a broad definition of incident; ITIL says "Failure of a configuration item that has not yet affected service is also an incident — for example, failure of one disk from a mirror set. The ITIL incident management process ensures that normal service operation is restored as quickly as possible and the business impact is minimized."
```

Comment:
I tried looking it up and was really a bit confused, but at least I'll put in a link to https://en.wikipedia.org/wiki/ITIL :)